### PR TITLE
Added MyAnonamouse tracker

### PR DESCRIPTION
**Pull request by coolius**
Issue #161

### CODE REVIEW

File: src/Jackett/Jackett.csproj [hunk 1]
```diff
@@ -444,6 +445,7 @@
     <Content Include="Content\login.html">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Content Include="Content\logos\myanonamouse.png" />
```
Comment (JigSawFr) [inline src/Jackett/Jackett.csproj]: Error in your code:
Must include _Copy to Output Directory instruction_

See after for usage ;)

**Comment by coolius** [comment]
Added CopytoOutputDirectory setting to logo.

**Comment by JigSawFr** [comment]
Thanks you, under review.

**Comment by flightlevel** [comment]
Thanks @coolius , your changes are in the current release

**Pull request merged**